history/characters/1_azerothian.txt

```diff
@@ -7365,32 +7365,56 @@
 #dynasty=108 (Lothar)
 # Thoradin
 1005={
-	name=Thoradin
-	# dynasty=108
-	culture=arathorian religion=old_ways
-	martial=5 diplomacy=6 stewardship=8 intrigue=5 learning=7
-	trait=education_martial_4 trait=brave trait=ambitious trait=just trait=strategist
+	name = Thoradin
+	dynasty = 16110
```

Comment:
`error.log` reports it can't find this dynasty.
```
[17:02:17][characterhistory.cpp:821]: Failed to read key reference 16110 from database  file: history/characters/1_azerothian.txt line: 7928
```

Reply:
I can't find it, it seems to me already fixed